src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.security.SecureRandom;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private final int randomNumber = new SecureRandom().nextInt(Color.values().length);
```

Comment:
The randomNumber of instances of this class is hardcoded, so it's impossible to reuse them. It's better to get a random number on each getRandomColor() call.